scout/adapter/mongo/variant.py

```diff
@@ -860,12 +860,13 @@ def get_region_vcf(
 
         return file_name
 
-    def case_variants_count(self, case_id, institute_id, update_case=True):
+    def case_variants_count(self, case_id, institute_id, variant_type=None, update_case=True):
         """Returns the sum of all variants for a case by type
 
         Args:
             case_id(str): _id of a case
             institute_id(str): id of an institute
+            variant_type(str): "clinical" or "research"
             update_case(bool): whether the case document should be updated with these stats
```

Comment:
I'm not a big fan of this variable name any more, at least not with the current functionality. `force_update_case`?

Reply:
Sorry I haven't understood the question

Reply:
Ahhh Now I get it, I was thinking the variable up, variant_type. Yes let's rename it!

Reply:
sry, I hit the wrong line - but exactly!